R/DBConnection.R

```diff
@@ -106,6 +106,103 @@ setGeneric("dbSendQuery",
   valueClass = "DBIResult"
 )
 
+#' Execute a non-select query
```

Comment:
@imanuelcostigan if this isn't a query, what is it? Maybe "Execute a statement that doesn't return values" ?